python/cudf/cudf/core/_base_index.py

```diff
@@ -1432,6 +1432,61 @@ def rename(self, name, inplace=False):
             out.name = name
             return out
 
+    def indices_of(self, value) -> cudf.core.column.NumericalColumn:
```

Comment:
Do these need to be private (`_`) methods?

Reply:
I suppose the `Column.indices_of` method can be "public", since I suppose that is not a stable API. But yes, probably the user-facing `Index` methods should be private.

Reply:
Done.